west.yml

```diff
@@ -206,12 +206,12 @@ manifest:
           compare-by-default: false
     - name: sidewalk
       repo-path: sdk-sidewalk
-      revision: v2.9.0-rc1
+      revision: v2.9.0-rc2
       groups:
         - sidewalk
     - name: find-my
       repo-path: sdk-find-my
-      revision: 0170d0374ffe379522a0d2eaf5b44d7d1aa1e7b7
+      revision: v2.9.0-rc2
```

Comment:
Incorrect tag. The v2.9-branch should have been used for tagging in the sdk-find-my repository

Reply:
I fixed this by force-pushing the tag aliases in the `sdk-find-my` repository.